libr/bin/bin.c

```diff
@@ -124,6 +124,12 @@ R_API void r_bin_options_init(RBinOptions *opt, int fd, ut64 baseaddr, ut64 load
 	opt->rawstr = rawstr;
 }
 
+R_API void r_bin_arch_options_init(RBinArchOptions *opt, const char *arch, int bits) {
+	memset (opt, 0, sizeof (*opt));
```

Comment:
you are setting all the fields of the struct, no need for a memset

Reply:
ok